src/main/java/hudson/plugins/analysis/core/AbstractResultAction.java

```diff
@@ -104,12 +106,12 @@ public ToolTipProvider getToolTipProvider() {
      *
      * @return the associated build of this action
      */
-    public final AbstractBuild<?, ?> getOwner() {
+    public final Run<?, ?> getOwner() {
```

Comment:
@oleg-nenashev the plugin maintainer is responsible for determining the version number strategy.

The general convention in Jenkins Plugin land has been to bump the major version **if the configuration file format is no longer backwards compatible**.

This type of change does not affect the configuration files, so unless @uhafner feels this should be a major version bump or unless @uhafner has provided a CONTRIBUTING.md that defines a version number policy I do not see this as needing a major version bump.

In any case there is an annotation available to all Jenkins plugins, `WithBridgeMethods` that removes the binary compatibility issue